modular_skyrat/modules/modular_implants/code/nifsoft_shop.dm

```diff
@@ -0,0 +1,34 @@
+/obj/machinery/vending/nifsoft
+	name = "NIFSoft Vendor"
+	desc = "Money can be exchanged here for NIFSofts and other NIF goods" //Placeholder description
+	icon = 'modular_skyrat/modules/modular_implants/icons/obj/machines/vendors.dmi'
+	icon_state = "proj"
+	density = FALSE
```

Comment:
why

Reply:
(they're holograms, that you can only see with an NIF)